src/ResourceManager/DataFactories/Commands.DataFactoryV2/DataFactoryContextBaseCmdlet.cs

```diff
@@ -35,5 +36,14 @@ public abstract class DataFactoryContextBaseCmdlet: DataFactoryBaseCmdlet
             HelpMessage = Constants.HelpFactoryName)]
         [ValidateNotNullOrEmpty]
         public string DataFactoryName { get; set; }
+
+        protected void ByFactoryObject(PSDataFactory dataFactory)
```

Comment:
@hvermis why does `dataFactory` need to be passed in if you have a reference to `DataFactory` above?

Reply:
Refactoring issues - was trying to make more generic for other cmdlets too that have PSDatafactory called InputObject, and hit other issues. Removed it.